style.css

```diff
@@ -419,14 +419,177 @@ input:focus {
   border-color: rgb(98, 108, 201);
 }
 
+.headerlish {
+  display: none;
+}
+
 @media (min-width: 375px) {
   .content-section {
     background-size: cover;
   }
 }
 
+/* desktop version */
+
 @media (min-width: 768px) {
+  .icon-Union {
+    display: none;
+  }
+
+  .topBar-nav {
+    position: sticky;
+    top: 0;
+    background-color: white;
+  }
+
+  .headerlist {
+    display: flex;
+    gap: 8px;
+    list-style-type: none;
+    padding-right: 20px;
+    font-family: var(--primary-font);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+    position: sticky;
+    top: 0;
+  }
+
+  .headerlist a:hover {
+    color: rgb(24, 155, 160);
+  }
+
   .content-section {
+    background-image: url(img/Header\ bg-desktop.png);
+    background-size: contain;
+    display: flex;
+    height: 718px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .main-Container {
+    width: 60%;
+    margin-left: 0;
+    padding-top: 0;
+  }
+
+  .work-container {
+    display: flex;
+    margin: 30px 10px;
+  }
+
+  .cards {
+    width: auto;
+    flex-direction: row;
+    gap: 25px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .snapshot-portofolio img {
+    padding-bottom: 0;
+  }
+
+  .blocks {
+    gap: 10px;
+  }
+
+  .tframes {
+    gap: 12px;
+  }
+
+  .tframes h3 {
+    font-weight: 500;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .frames-list {
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .reverse-block {
+    flex-direction: row-reverse;
+    gap: 20px;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .project-text {
+    align-items: center;
+    font-size: 16px;
+    width: 100%;
+  }
+
+  .about-section {
+    display: flex;
+    height: auto;
+    gap: 12px;
+    flex-direction: row;
     width: 80%;
+    justify-content: center;
+    margin: 0;
+  }
+
+  .aboutWrapper {
+    padding-top: 10px;
+    width: 40%;
+  }
```

Comment:
- [ ] Great job with the about me section :+1:, but to match the Figma template, please consider the following recommendations:
- Center the section on the page to match the template.
- Round the top right corner of the section as shown in the template.

**Your design**

![Screenshot from 2023-04-01 16-17-38](https://user-images.githubusercontent.com/105117832/229314385-cf09ecba-c9bd-4ddd-8eab-fc7bae584414.png)

**Figma**

![Screenshot from 2023-04-01 16-17-18](https://user-images.githubusercontent.com/105117832/229314394-9a3ef7b9-c581-45f2-92f0-9445b573b196.png)